docs/source/en/tutorials/inference_with_big_models.md

```diff
@@ -0,0 +1,71 @@
+<!--Copyright 2024 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+-->
+
+# Working with big models
+
+A modern diffusion model ([SDXL](../using-diffusers/sdxl.md), for example) is not just a single model but a collection of multiple models. SDXL has four different model-level components:
+
+* A variational autoencoder (VAE)
+* Two text encoders
+* A denoiser (which has a UNet architecture)
+
+Usually, the text encoders and the denoiser are bigger and much larger in size compared to the VAE. 
+
+As models keep getting bigger and better, it’s possible your model is so big that even a single copy won’t fit in RAM. That doesn’t mean it can’t be loaded: if you have one or several GPUs, this is more memory available to store your model. In this case, it’s better if your model checkpoint is split into several smaller files that we call checkpoint shards.
+
+When a text encoder checkpoint has multiple shards ([T5-xxl for SD3](https://huggingface.co/stabilityai/stable-diffusion-3-medium-diffusers/tree/main/text_encoder_3), for example), it will be automatically handled by the `transformers` library as it is a required dependency of Diffusers when using the [`StableDiffusion3Pipeline`]. 
+
+The denoiser checkpoint can also have multiple shards and performing inference with such a checkpoint is supported in Diffusers thanks to the Accelerate library. 
```

Comment:
```suggestion
The UNet can also have multiple shards and supports inference thanks to the [Accelerate](https://huggingface.co/docs/accelerate/index) library.
```

Reply:
Stable Diffusion 3 doesn't have a UNet as its denoiser. It uses a Transformer backbone.